### DISCUSSION TITLE
Topics and speakers for OSSF London Oct 2021

### DISCUSSION THREAD

**Discussion by claredillon** [finos/InnerSource#21] [General]
OSSF London is happening on Oct 4-5 2021, alongside OSPOCon Europe (Oct 6th).

It's been proposed the FINOS InnerSource SIG submit one or more talks/panel sessions. 

CFP closes July 11th. 

Please use this thread to suggest topics or volunteer / nominate speakers.

**Comment by godds** [comment]
Panel discussion on "The state of InnerSource in banks" (or some such), with panellists from financial services organisations sharing, for example, challenges they face around InnerSource, their level of maturity, evidence of efficiencies/positives that have come from this approach.

(Obviously reliant on finding 2-3 panellists who are able to share their stories)

**Reply by gyehuda** [reply]
I'm interested in either sitting on a panel or presenting a version of my recent blog-series about culture, behaviors, and InnerSource within a bank environment.

**Comment by claredillon** [comment]
From an offline conversation with @richardwagener365 and building on @godds  suggestion above, here is a proposal for the theme/abstract. Appreciate feedback/comments/improvement

**InnerSource in Financial Services**

Panel session / 45 minutes.

Abstract:
InnerSource, the use of open source methods and practices to build proprietary code within an organization's firewall, has seen huge growth in Financial Services in the past year. A FINOS InnerSource Special Interest Group was formed in early 2021 to provide a forum to discuss challenges and share experiences and best practices. In this panel session we will get first hand feedback from the FINOS InnerSource SIG participants on why their organizations have adopted InnerSource, results so far, how to get started and how to foster engagement in FS InnerSource communities. The panel will also discuss what have been the biggest barriers in adopting InnerSource and how they overcame them.

**Comment by claredillon** [comment]
We had some good news and some bad news re OSSF this week. Our panel submission for OSSF London was accepted – hooray! Unfortunately, the event organizers confirmed they are only accepting in-person presenters, and many of our volunteer panelists cannot travel in person (the original submission allowed for virtual participants). 

If you are planning to attend OSSF London (Oct 4th/5th) and would be willing to discuss your InnerSource experiences on a panel – please do comment here or contact @claredillon . As an added incentive - note that panel participants get free entry to the event.

**Comment by claredillon** [comment]
OSSF Events Team have confirmed we can pre-record the panel session - so will be doing that! :) Any topics anyone thinks should definitely be covered - please feel free to add into this thread!

**Reply by toshaellison** [reply]
@claredillon Is it possible to have anyone available in-person in either NY or London to answer questions following the pre-recorded session? This will make a difference in terms of where it gets played and recording deadlines.
